src/ferui-components/tree-view/tests/tree-view-client.spec.ts

```diff
@@ -0,0 +1,93 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
```

Comment:
Do not create two spec file to test client and server treeview. Create one spec file by component file and cover both client and server within those files.

Reply:
Fixed. I moved all tests to one file now